### PR TITLE
[ko] Translated the glossary term 'Endpoints' into Korean

### PR DESCRIPTION
**Pull request by rollony**
from #35555

### CODE REVIEW

File: content/ko/docs/reference/glossary/endpoint.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: 엔드포인트(Endpoints)
+id: endpoints
+date: 2020-04-23
+full_link: 
+short_description: >
+  엔드포인트는 파드의 IP 주소와 일치하는 서비스 셀렉터를 추적한다.
```
Comment (jihoon-seo) [inline content/ko/docs/reference/glossary/endpoint.md:7]: [원문]
> Endpoints track the IP addresses of Pods with matching Service selectors.

원문에 따라 조금 수정해 보았습니다. 😊

```suggestion
  엔드포인트는 서비스(Service) 셀렉터에 매치되는 파드의 IP 주소를 추적한다.
```

File: content/ko/docs/reference/glossary/endpoint.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- networking
+---
+ 엔드포인트는 파드의 IP 주소와 일치하는 {{< glossary_tooltip term_id="selector" >}}를 추적한다.
```
Comment (jihoon-seo) [inline content/ko/docs/reference/glossary/endpoint.md:13]: 위의 제안과 일관성을 맞추기 위함입니다. 😊

```suggestion
 엔드포인트는 서비스(Service) {{< glossary_tooltip term_id="selector" >}}에 매치되는 파드의 IP 주소를 추적한다.
```

File: content/ko/docs/reference/glossary/endpoint.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+<!--more-->
+셀렉터를 명시할 필요 없이 {{< glossary_tooltip term_id="service" >}} 엔드포인트를 수동으로 설정할 수 있다.
+{{< glossary_tooltip term_id="endpoint-slice" >}} 리소스는 엔드포인트의 대안으로, 확장성(Scalable)과 확장 가능성(Extensible)을 제공한다.
```
Comment (jihoon-seo) [inline content/ko/docs/reference/glossary/endpoint.md:17]: [원문]

> Endpoints can be configured manually for {{< glossary_tooltip text="Services" term_id="service" >}} without selectors specified.
The {{< glossary_tooltip text="EndpointSlice" term_id="endpoint-slice" >}} resource provides a scalable and extensible alternative to Endpoints.

원문에 따라 조금 수정해 보았습니다. 😊

```suggestion
엔드포인트는 셀렉터를 명시하지 않고도 {{< glossary_tooltip term_id="service" >}}에 대해 수동으로 구성할 수 있다.
{{< glossary_tooltip text="엔드포인트슬라이스(EndpointSlice)" term_id="endpoint-slice" >}} 리소스는 엔드포인트의 대안으로, 확장성(Scalable)과 확장 가능성(Extensible)을 제공한다.
```

**Review by jihoon-seo — COMMENTED** [review]
@rollony 기여에 감사드리며, 몇 가지 의견을 남겼습니다.
확인 부탁드립니다. 😊
/assign

**Review by sftim — COMMENTED** [review]
A tip

I would make it clear that the `kind` for an Endpoints is `Endpoints` (plural) and that you cannot omit the “s” from the end. Readers who speak Korean and also know basic English grammar might assume otherwise.

File: content/ko/docs/reference/glossary/endpoint.md (hunk 3 above)
Comment (sftim) [reply content/ko/docs/reference/glossary/endpoint.md:17]: Does it make sense to write “scalable” and “extensible” in all-lowercase?
It's OK to omit the English originals if Korean natively has a way to explain, eg, that an API was designed to work at scale.

**Comment by rollony** [comment]
> A tip
> 
> I would make it clear that the `kind` for an Endpoints is `Endpoints` (plural) and that you cannot omit the “s” from the end. Readers who speak Korean and also know basic English grammar might assume otherwise.

Thank you for the tip!

In the Korean language we do not distinguish between plural and singular nouns in formal context.
But I will leave the original English term in the title along with the translated Korean term, so readers can see that the term is supposed to be plural.

**Comment by sftim** [comment]
It's also OK to write a short note about the pluralness (or explicitly say something like 이 API의 이름은 Endpoint가 아니라 Endpoints입니다).

**Comment by rollony** [comment]
> It's also OK to write a short note about the pluralness (or explicitly say something like 이 API의 이름은 Endpoint가 아니라 Endpoints입니다).

Okay, that seems like a more convenient option for readers to understand.
I will make sure to add a note that serves the purpose. Thanks!

**Comment by rollony** [comment]
@sftim 

I noticed that the names of the glossary markdown files are 'endpoint.md' instead of 'endpoints.md', even in the original English version.
Should this be an issue to be addressed?

File: content/ko/docs/reference/glossary/endpoint.md (hunk 3 above)
Comment (rollony) [reply content/ko/docs/reference/glossary/endpoint.md]: > Does it make sense to write “scalable” and “extensible” in all-lowercase? It's OK to omit the English originals if Korean natively has a way to explain, eg, that an API was designed to work at scale.

Thanks for the comment, I made some changes reflecting your comment and to better fit the sentence in the translated Korean context.

**Comment by rollony** [comment]
@jihoon-seo 
리뷰해주셔서 감사합니다!
말씀하신 사항들 반영하여 수정해보았습니다.

**Comment by jihoon-seo** [comment]
@rollony @sftim Thank you all!
/lgtm

File: content/ko/docs/reference/glossary/endpoint.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+date: 2020-04-23
+full_link: 
+short_description: >
+  엔드포인트는 서비스(Service) 셀렉터에 매치되는 파드의 IP 주소를 추적한다.
+
+aka:
+tags:
+- networking
+---
+ 엔드포인트는 서비스(Service) {{< glossary_tooltip term_id="selector" >}}에 매치되는 파드의 IP 주소를 추적한다. (이 API의 원문 영어 이름은 'Endpoint'가 아니라 'Endpoints'이다.)
```
Comment (seokho-son) [inline content/ko/docs/reference/glossary/endpoint.md:13]:
```suggestion
 엔드포인트는 서비스(Service) {{< glossary_tooltip term_id="selector" >}}에 매치되는 파드의 IP 주소를 추적한다. (API에서 해당 `kind`의 명칭은 `Endpoints` 이며, 복수형이 기본임을 유의한다.)
```
정도로 표시하면 어떨까요?
꼭 반영하실 필요는 없으며, 현재 상태로도 

/lgtm

입니다. :)

**Comment by rollony** [comment]
@seokho-son 

말씀하신 사항 반영해 보았습니다!
리뷰해주셔서 감사합니다 :)

**Comment by seokho-son** [comment]
감사합니다! :)

/lgtm
/approve

**Pull request merged**